src/test/storm_gtest.h

```diff
@@ -27,10 +27,19 @@ inline GTEST_API_ AssertionResult DoubleNearPredFormat(const char* expr1, const
 
 namespace storm {
 namespace test {
-inline void initialize() {
+extern bool noGurobi;
+
+inline void initialize(int* argc, char** argv) {
```

Comment:
I had to lookup how gtest arguments are handled. Maybe add a comment:
`// GoogleTest-specific commandline arguments were already processed before and removed from argc/argv`